net-misc/sstp-client/sstp-client-1.0.10-r1.ebuild

```diff
@@ -38,8 +38,8 @@ src_prepare() {
 
 src_configure() {
 	local PPPD_VER="$(best_version net-dialup/ppp)"
-	PPPD_VER=${PPPD_VER#*/*-} #reduce it to ${PV}-${PR}
-	PPPD_VER=${PPPD_VER%%[_-]*} # main version without beta/pre/patch/revision
+	PPPD_VER=${PPPD_VER#*/*-} 		# reduce it to ${PV}-${PR}
+	PPPD_VER=${PPPD_VER%%[_-]*}		# main version without beta/pre/patch/revision
```

Comment:
indentation should be tabs, what you do for comments is your business... and doign tabs here is absolutely awful